web-server/src/content/PullRequests/DeploymentInsightsOverlay.tsx

```
@@ -267,170 +293,215 @@ export const DeploymentInsightsOverlay = () => {
-                      {loadingPrs ? (
-                        <MiniCircularLoader label="Loading pull requests..." />
-                      ) : prs.length ? (
-                        <PullRequestsTableMini
-                          prs={prs}
-                          updateSortConf={updateSortConf}
-                          conf={conf}
-                          hideColumns={hideTableColumns}
-                        />
+                        {deployments.length}
+                      </Line>{' '}
+                      {Boolean(failedDeps.length) ? (
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)